### ISSUE TITLE
add support for showing window title in fullscreenmode

### ISSUE THREAD

**Issue by jose1711** [derf/feh#510]
on my 3 monitor configuration it's sometimes difficult to see whether `feh`'s fullscreen window is in focus (hence receiving shortcuts) or not. the ability to see the window title would definitely help here. currently though with `-F` switch the whole screen is filled and there is no option to make the window title appear.

**Comment by Ferada** [comment]
Wouldn't `feh --draw-filename -F` be enough?

**Comment by jose1711** [comment]
Nope, because the filename itself does not give me the "in-focus/out-focus" information while window title typically changes color/tint.

**Comment by derf** [comment]
Isn't your window manager able to make your feh window fullscreen? cause that seems a lot like a job for a window manager and not an individual application

**Comment by jose1711** [comment]
I am using `metacity` (`GNOME`) and AFAIK it does not offer any integrated means of clients' windows manipulation.

**Comment by jose1711** [comment]
Furthermore there does not seem to be a way to let window manager handle the window (e. g. toggle titlebar, resize window etc) and at the same time use autozoom's feature of `feh` (as autozoom only works for fixed geometry/fullscreen and is not recalculated upon changing the window dimensions).

`feh -Z` adds titlebar but zoom must be taken care of manually.
